src/wrapper/AirtopSessions.ts

```diff
@@ -31,14 +31,14 @@ export class AirtopSessions {
   list(
     request?: Airtop.SessionsListRequest,
     requestOptions?: SessionsNamespace.RequestOptions,
-  ): Promise<Airtop.SessionsWithPaginationEnvelopeDefaultMetaWrapper> {
+  ): Promise<Airtop.SessionsResponse> {
```

Comment:
I was wondering if we should just remove the types and let typescript deduct them from whatever is returned from sessions.list in this example.

What do you think?

Reply:
That would be better for maintenance here, for sure.